### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by Stevengrant**

### CODE REVIEW

File: code401challenges/src/test/java/code401challenges/LibraryTest.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class LibraryTest {
+    @Test public void testSomeLibraryMethod() {
+        Library classUnderTest = new Library();
+        assertTrue("someLibraryMethod should return 'true'", classUnderTest.someLibraryMethod());
+    }
+    @Test public void test_insertShiftArray(){
```
Comment (Tw1stedRain) [inline code401challenges/src/test/java/code401challenges/LibraryTest.java]: no points off this time, but in the future, I'd like to see more tests, specifically some edge cases. Ex. what would happen if the given was an epmty array, odd vs. even array length, a couple different examples. I want to know that it doesn't just work with this one test, but not others
